src/sentry/static/sentry/app/views/performance/landing/utils.tsx

```diff
@@ -72,6 +96,41 @@ export function getBackendFunction(
   }
 }
 
+const VITALS_FRONTEND_PLATFORMS: string[] = [...frontend];
+const VITALS_BACKEND_PLATFORMS: string[] = [...backend];
+
+export function getDefaultDisplayFieldForPlatform(
+  eventView: EventView,
+  projects: Project[]
+) {
+  if (!eventView) {
+    return LandingDisplayField.ALL;
+  }
+  const projectIds = eventView.project;
+  if (
+    projectIds.length > 1 ||
```

Comment:
Can be a followup but should this account for the fact that it's possible to select multiple frontend projects together?

Reply:
I'll do it now